### PR TITLE
Bump up Yorkie to v0.3.0

### LINKED ISSUE #54: Apply new spec of Counter to documentation
**Description**:
Although Counter API is changed in yorkie-team/yorkie#441 and  yorkie-team/yorkie-js-sdk#426, Counter usage example in SDK documentation is not updated yet:

https://github.com/yorkie-team/yorkie-team.github.io/blob/34e382b81029c8cfb865bc549bdfe8a4cdd884b8/docs/js-sdk.mdx?plain=1#L208-L215

The example should be updated after determining the order of constructor arguments. (Refer yorkie-team/yorkie-js-sdk#447.)

### PR DESCRIPTION
**Pull request by cozitive**
#### What this PR does / why we need it?
Bump up Yorkie to 0.3.0
- SDK documentation should reflect Text-Richtext merge and Counter interface change.

#### What are the relevant tickets?

Fixes #54

### Checklist
- [x] Added relevant tests or not required
- [x] Didn't break anything

### CODE REVIEW

**Comment by chacha912** [comment]
Looks good! 

Could you also update the example code on the Products page?
image

File: MAINTAINING.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Maintaining
+
+## Releasing a New Version
+
+### Updating and Deploying homepage
+
+1. Update `version` in `package.json`.
```
Comment (hackerwins) [inline MAINTAINING.md]:
```suggestion
1. Update `version` in [package.json](https://github.com/yorkie-team/yorkie-team.github.io/blob/main/package.json#L36).
```

File: MAINTAINING.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+2. Update `NEXT_PUBLIC_YORKIE_VERSION` and `NEXT_PUBLIC_YORKIE_JS_VERSION` in `.env`.
```
Comment (hackerwins) [inline MAINTAINING.md]:
```suggestion
2. Update `NEXT_PUBLIC_YORKIE_VERSION` and `NEXT_PUBLIC_YORKIE_JS_VERSION` in [.env](https://github.com/yorkie-team/yorkie-team.github.io/blob/34e382b81029c8cfb865bc549bdfe8a4cdd884b8/.env#L2-L3).
```

**Review by hackerwins — APPROVED** [review]
Thanks for your contribution.

**Pull request merged**